packages/core/src/Stores/active-symbols-store.js

```diff
@@ -0,0 +1,17 @@
+import { WS } from '@deriv/shared';
```

Comment:
add this store to separate active_symbols observable from trader store and can use it in core and reports stores